frontend/src/components/auth/Profile.jsx

```diff
@@ -78,6 +81,9 @@ const Profile = () => {
             if (interviewQuestions.length > 2) {
                 setLastQuestionCheck(interviewQuestions[currentQuestionIndex + 2])
             }
+
+            const isLastQuestion = currentQuestionIndex === interviewQuestions.length -2;
```

Comment:
would currentQuestionIndex ever be greater than interviewQuestions.length -2? would that still mean it's the last question?